docs/source/agent.rst

```diff
@@ -127,3 +127,23 @@ game is finished and one participant withdraws the stake, the claim will move in
 state.
 
 If an agent is not participating in a claim, that claim will transition into the ``ignored`` state.
+
+.. _Unsafe Fill Time:
+
+Unsafe Fill Time
+----------------
+
+Agent cli command has an option ``--unsafe-fill-time`` to configure. With this option, agent can 
+decide which requests to ignore looking at request validity period. This option added to lower the risk 
+of filling a request which is too close to expire.
+
+.. graphviz:: unsafe_fill_time.dot
+   :align: center
+   :caption: Unsafe fill time
+
+In the graph, T1 is representing the request creation time, T2 is calculated using ``unsafe-fill-time`` option, and T3 is the request expiration.
+
+If unsafe time increases, T2 moves to left, so agent can process more requests.
```

Comment:
I think this should be
```
If unsafe time increases, T2 moves to left, so the agent will fill fewer requests.
```

(The safe window decreases so we can fill fewer requests.)